src/Image.js

```diff
@@ -0,0 +1,75 @@
+import Modal from 'react-responsive-modal';
+import 'react-responsive-modal/lib/react-responsive-modal.css';
+
+import { PropTypes as T } from 'prop-types';
```

Comment:
I really dislike the `PropTypes as T` thing. your editor will autocomplete PropTypes when typing it, and minification handles size. Theres no reason you shouldn't be specific